modules/nodes-secondary-scheduler-remove-resources-console.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-uninstalling.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-secondary-scheduler-remove-resources-console_{context}"]
+= Removing {secondary-scheduler-operator} resources
+
+Optionally, after uninstalling the {secondary-scheduler-operator-full}, you can remove its related resources from your cluster.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Remove CRDs that were installed by the {cert-manager-operator}:
```

Comment:
This attribute is missing in the rendered output, I'm not sure if this is expected or not so just making a note of it 🙂